stripe/src/main/java/com/stripe/android/model/Source.java

```diff
@@ -185,6 +185,7 @@ private Source(
         mUsage = usage;
     }
 
+    @Nullable
```

Comment:
Several other getter and setting methods could be updated with `@Nullabel` annotations.